modules/300-prometheus/crds/prometheusremotewrite.yaml

```diff
@@ -56,6 +56,13 @@ spec:
                 customAuthToken:
                   type: string
                   description: Custom token sent as a value of the X-Auth-Token header.
+                headers:
```

Comment:
Let's replace the current `headers` definition to make it more flexible and scalable for future header additions without modifying the schema each time.

```yaml
headers:
  type: object
  patternProperties:
    "^X-Scope-OrgID$": 
      type: string
  additionalProperties: false  # Prevents undefined headers
```

This allows easier extension for new headers (`X-Auth`, etc.) with the same validation rules while keeping flexibility.

Reply:
done